.env.example

```diff
@@ -3,3 +3,5 @@ NGROK_HOSTNAME=
 NGROK_PORT=bot:8080
 SLACK_TOKEN=xoxb-...
 DB_FILE=db/phpmxbot.db
+ENVIRONMENT=prod
+DEBUG=false
```

Comment:
Is this used somewhere?

Reply:
Hey good morning,  it's using inside of the definition of the container https://github.com/phpmx/phpmxbot/pull/37/files#diff-c47b03734cd2b4337b345eeba955637ba790d51fd98979f6d366d4acbdb104e0R46